internal/proto/data_coord.proto

```diff
@@ -405,6 +405,10 @@ message Binlog {
   string log_path = 4;
   int64 log_size = 5;
   int64 logID = 6;
+  // memory_size represents the size occupied by loading data into memory.
+  // log_size represents the size after data serialized.
+  // for stats_log, delta_log, the memory_size always equal log_size.
```

Comment:
delta_log's MemSize should be different to LogSize, since it also has a very large compression ratio.
Please add deltalog's MemSize accordingly

Reply:
it‘s already done, I will modify the comment.